dev-qt/qtwayland/qtwayland-5.15.3.9999.ebuild

```diff
@@ -27,7 +27,9 @@ DEPEND="
 		x11-libs/libXcomposite
 	)
 "
-RDEPEND="${DEPEND}"
+RDEPEND="${DEPEND}
+	=dev-qt/qtwaylandscanner-${QT5_PV}*:5
```

Comment:
We could use the [5.15.3 bump](https://github.com/gentoo/qt/pull/251) as a way to communicate this dependency being dropped.

Reply:
Dropped dependency and will make it part of the initial package.mask message and/or gentoo-dev mail.